doc/spec/pantry-spec.md

```diff
@@ -0,0 +1,66 @@
+# Roux Pantry Spec
+
+Ingredients in the Roux ecosystem are distributed in a **pantry**: a filesystem
+subtree containing one or more ingredients.
+
+For example:
+
+```
+┌── my-project
+│   └── pantry
+│       ├── forms
+│       │   └── index.scss
+│       │   └── index.hbs
+│       │   └── ingredient.md
+│       ├── sidebar
+│       │   └── index.scss
+│       │   └── index.hbs
+│       │   └── ingredient.md
+│       │   └── model.js
+```
+
+Pantries are typically named `pantry` but can be named anything you choose.
```

Comment:
Maybe a better way to put it is "Pantry folders are typically named pantry" — the actual folder itself. I was trying to note that you can have your pantry folder be anything you want; pantry, docs, myspecialsnowflakepantryfolder, whatever. Then when you configure it you pass that path, `whatever/pantry` as the path. You're right to be more clear that the _name_ and the _folder name_ of the pantry are different.